cluster/juju/util.sh

```diff
@@ -30,6 +30,8 @@ source "${JUJU_PATH}/${KUBE_CONFIG_FILE-config-default.sh}"
 source ${JUJU_PATH}/prereqs/ubuntu-juju.sh
 export JUJU_REPOSITORY=${JUJU_PATH}/charms
 KUBE_BUNDLE_PATH=${JUJU_PATH}/bundles/local.yaml
+# The destination directory for the kubectl binary file.
+KUBECTL_DIR=${KUBE_ROOT}/platforms/linux/amd64
```

Comment:
must be quoted (this whole file does a terrible job quoting bash variable, so lets start making them better)
`KUBECTL_DIR="${KUBE_ROOT}/platforms/linux/amd6"